sdk/storage/Azure.Storage.Blobs.Batch/src/ServiceRestClient.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using Azure.Core;
+
+namespace Azure.Storage.Blobs.Batch
+{
+    internal partial class ServiceRestClient
+    {
+        // We are overriding this method because the new generator attempts to add 2 content types to the Content-Type header,
+        // causing auth to fail.  https://github.com/Azure/autorest/issues/3914
+        internal HttpMessage CreateSubmitBatchRequest(long contentLength, string multipartContentType, Stream body, int? timeout)
```

Comment:
same here. plz reference issue created in our repo in the comments.

Reply:
fixed.